frontend/src/js/components/auditlogs/EventDetailsDrawerContentMap.jsx

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import DeviceConfiguration from './eventdetails/deviceconfiguration';
+import FileTransfer from './eventdetails/filetransfer';
+import PortForward from './eventdetails/portforward';
+import TerminalSession from './eventdetails/terminalsession';
+import { UserChange } from './eventdetails/userchange';
+
+const changeTypes = {
+  user: 'user',
+  device: 'device',
+  tenant: 'tenant'
+};
+
+const configChangeDescriptor = {
+  set_configuration: 'definition',
+  deploy_configuration: 'deployment'
+};
+
+const EventDetailsDrawerContentMap = (item, FallbackComponent = null) => {
```

Comment:
This is still a mapping utility masquerading as a Component 😉. Personally I would also do 
```suggestion
import EventDetailsFallbackComponent from './eventdetails/FallbackComponent';

const EventDetailsDrawerContentMap = (item, FallbackComponent = EventDetailsFallbackComponent) => {
```
to default to this being the Fallback